### PR TITLE
Bring back health check

### PR DESCRIPTION
**Pull request by CCLDArjun**

### CODE REVIEW

**Review by neelp03 — CHANGES_REQUESTED** [review]
Linting checks failed

File: api/peripheral_api/routes/Printer.js [hunk 1]
```diff
@@ -27,7 +27,24 @@ AWS.config.update({
...
 
 router.get('/healthCheck', async (req, res) => {
-  return res.sendStatus(OK);
+/*
+ * How these work with Quasar:
+ * https://github.com/SCE-Development/Quasar/wiki/How-do-Health-Checks-Work%3F
+ */
+  if (process.env.NODE_ENV !== 'production') {
+    return res.sendStatus(OK);
+  }
```
Comment (neelp03) [inline api/peripheral_api/routes/Printer.js:36]: we dont use this anymore see PR #1109 for details

**Review by neelp03 — CHANGES_REQUESTED** [review]

**Review by neelp03 — CHANGES_REQUESTED** [review]
ALso change all backend dockerfiles to have an argument called NODE_ENV

1 line change in all docker files:
ARG NODE_ENV

File: api/peripheral_api/routes/Printer.js (hunk 1 above)
Comment (CCLDArjun) [reply api/peripheral_api/routes/Printer.js:36]: we still have `ARG NODE_ENV` though?

**Review by neelp03 — APPROVED** [review]
lgtm

**Review by evanugarte — APPROVED** [review]
as long as u tested this and it works

**Comment by neelp03** [comment]
I’ll test it again tmr

**Pull request merged**